website/templates/emails/pending_registration_admin.html.mako

```diff
@@ -11,16 +11,34 @@
     ${initiated_by} has initiated a registration of your project ${project_name}. The proposed registration can be viewed here: ${registration_link}.<br>
     % endif
     <br>
-    If approved, a registration will be created for the project and will be made public immediately.<br>
+
+    % if is_moderated:
+         If approved, a registration will be created for the project and will be made public immediately and sent to ${reviewable.provider.name} moderators for review
```

Comment:
"If approved, a registration will be created for the project ~~and will be made public immediately~~ and sent to ${reviewable.provider.name} moderators for review"